packages/shared/src/lib/constants.ts

```diff
@@ -39,6 +39,7 @@ export const squadsPublicSuggestion =
 export const squadsPublicGuide = 'https://r.daily.dev/public-squads-guide';
 export const searchFeedback = 'https://r.daily.dev/search-feedback';
 export const searchDocs = 'https://r.daily.dev/search-docs';
+export const submitPublicSquads = 'https://r.daily.dev/public-squads-guide';
```

Comment:
This is already defined in `squadsPublicGuide` above
```suggestion
```

Reply:
Oh, nice. Let me remove this.